### ISSUE TITLE
Upgrade SuiteCRM in our own language [RFC]

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#3103]
#### Issue

The SuiteCRM version upgrade pack uses an english language file in \scripts\en_us.lang.php
This file is translated on Crowdin for many language but not available on official upgrade packs. 

@mattlorimer Is there any plan to include translations for that file so we can upgrade using the same admin language?
If so, can you please create a Github Repository so I can commit files there?

#### Expected Behavior

Upgrade SuiteCRM using the same language the administrator is logged in.

#### Actual Behavior

Now only English available!

#### Possible Fix

Include translations from Crowdin https://crowdin.com/project/suitecrmtranslations or create a Github repository to get those translations.

#### Context

Non english users shouldn't have to work in different languages!

#### Your Environment

* SuiteCRM Version used: 7.8.1

**Comment by chris001** [comment]
@horus68 
There's not much more to do to get the languages working for upgrades, or even for new installs.
1. The application should provide a list of languages to the user to select, then download https://crowdin.com/download/project/suitecrmtranslations/pt-BR.zip (substitute your language code), or download https://crowdin.com/download/project/suitecrmtranslations.zip for all languages.
2. Unzip into the SuiteCRM application folder so that the translated language files end up in the correct folders corresponding to the English files, next to all of the `en_us.lang.php` files.

**Comment by horus68** [comment]
This is just for the upgrade paks, not for the full language pack.
English file is in  \scripts\en_us.lang.php 
Translations for that file needs to be packed inside the official update pack (one file for each language)

**Comment by chris001** [comment]
@horus68   Right.  It's not much difference in downloading full language pack for one language (500KB) on the fly, versus downloading that a single language file (10 KB) - the one language file used during the upgrade procedure - versus downloading on the fly all language packs for all (50?) languages (25 MB) - from crowdin.
I found a pre built PHP code package, available thru composer, which does this exact function, download (and upload) language packs and individual language files, from crowdin, for joomla.  Easily adaptable to SuiteCRM / SugarCRM.
https://packagist.org/packages/joomla/crowdin-sync

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Upgrade SuiteCRM in our own language [RFC]](https://trello.com/c/9TMoqKvk/28-upgrade-suitecrm-in-our-own-language-rfc)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
